stories/Molecules/Navbar/Breadcrumb/breadcrumb-link.scss

```diff
@@ -0,0 +1,81 @@
+@import '../../../assets/scss/breakpoints';
+@import '../../../assets/scss/variables';
+@import '../../../assets/scss/mixins';
+
+.breadcrumb__wrapper {
+  align-items: center;
+  display: flex;
+  flex-wrap: wrap;
+  font-size: 12px;
+  padding: 0;
+
+  li {
+    display: inline-block;
+    margin: 10px;
+    position: relative;
+
+    &::after {
+      @include translate(0,-50%);
+
+      background: url(#{$img-path}/Separator.svg) no-repeat left center;
+      content: '';
+      height: 10px;
+      position: absolute;
+      right: -11px;
+      top: 50%;
+      width: 4px;
+    }
+
+    span {
+      color: $color-black;
+      display: block;
+      font-family: $Font-Family-ProximaSemiBold;
+      overflow: hidden;
+      text-overflow: ellipsis;
+      text-transform: uppercase;
+      white-space: nowrap;
+      width: 60px;
+
+      @include devicebreak(medium) {
+        overflow: inherit;
+      }
+    }
+
+    &:last-child::after {
+      content: none;
+    }
+  }
+
+  &--Modifier {
+    background-color: $color-gray-400;
+
+    li {
+      &::after {
+        background: url(#{$img-path}/Separator_white.svg) no-repeat left center;
+      }
+
+      span {
+        color: $color-white;
+      }
+
+      a {
+        color: $color-white;
+
+        &:hover {
+          color: $color-white;
+        }
+      }
+    }
+  }
+}
+
+[dir='rtl'] {
+  .breadcrumb__wrapper li::after,
+  .breadcrumb__wrapper--Modifier li::after {
+    @include transform(translate(0,-50%) rotate(120deg));
+
+    left: -15px;
+    right: unset;
+    top: 50%;
+  }
+}
```

Comment:
@pallavikadam-srijan can you please create css sprite for separator images...take the help from harish if you need